### DISCUSSION TITLE
Occular tab FOV differs from screen FOV

### DISCUSSION THREAD

**Discussion by CraigHPrice** [Stellarium/stellarium#757] [Q&A] [answered]
### Actual Behaviour:
After selecting the ocular and telescope as above, bottom of screen shows 1.28 degrees FOV and the ocular view on screen is a 1.28 degree FOV as well.  When I turn off ocular view and manually scroll to .6654 degree FOV with my mouse, the screen looks exactly like my actual Celestron 8" view.  Note also that changing OCULAR tab to ocular #2 (Plossl 14mm) changes ocular tab FOV to correct .3445 but screen to .689 and screen view to consistent with the .689.  Looking at Jupiter is most revealing of this problem.

### Steps to reproduce:
Pretty well described above.

### System
* Stellarium version: 0.19.1.1
* Operating system: Windows 10, version 1809
* Graphics Card: Intel HD Graphics 530 version 23.20.16.4973

### Logfile 
N/A

**Comment by gzotti** [comment, marked as answer]
Please read section 15.1.2 of the Stellarium User Guide, esp. the effect of the "Scale image circle" checkbox.

**Comment by CraigHPrice** [comment]
Thanks.  That was the issue.  Not sure what the intent of the "scale image circle" function is, but seems like it should be "Off" by default on installation, or flagged more clearly when on.  Thanks for the quick reply.

**Comment by gzotti** [comment]
The idea is to compare oculars with different apparent fields of view. Please flip a few pages and read section 15.1.3.
